docs/usecases/datastorage_for_institutions.rst

```diff
@@ -128,25 +163,254 @@ where the data are located.
+use the :command:`datalad create-sibling-ria` command to establish a connection
+to an either pre-existing or not-yet-existing RIA store, publish dataset contents
+with :command:`datalad publish`, (let others) clone the dataset from the
+RIA store, and (let others) publish and pull updates. In the
+case of large, institute-wide datasets, the RIA store can serve as a singular,
+central storage location that enables fine-grained data access to everyone who
+needs it, and as a storage and back-up location for all analyses datasets.
+
+The layout of a RIA store is a tree of datasets. The first level of subdirectories
+in this tree consists of the first three characters of the :term:`dataset ID`,
+and the second level of subdatasets contains the remaining characters of the
+dataset ID. Thus, the first two levels of subdirectories in the tree are split
+dataset IDs of the datasets that are stored in them [#f1]_. The code block below
+illustrates how a single DataLad dataset looks like in a RIA store, and the
+dataset ID of the dataset (``946e8cac-432b-11ea-aac8-f0d5bf7b5561``) is
+highlighted:
+
+.. code-block::
+   :emphasize-lines: 2-3
+
+    /path/to/my_riastore
+    ├── 946
+    │   └── e8cac-432b-11ea-aac8-f0d5bf7b5561
+    │       ├── annex
+    │       │   └── objects
+    │       │       ├── 6q
+    │       │       │   └── mZ
+    │       │       │       └── MD5E-s93567133--7c93fc5d0b5f197ae8a02e5a89954bc8.nii.gz
+    │       │       │           └── MD5E-s93567133--7c93fc5d0b5f197ae8a02e5a89954bc8.nii.gz
+    │       │       ├── 6v
+    │       │       │   └── zK
+    │       │       │       └── MD5E-s2043924480--47718be3b53037499a325cf1d402b2be.nii.gz
+    │       │       │           └── MD5E-s2043924480--47718be3b53037499a325cf1d402b2be.nii.gz
+    │       │       ├── [...]
+    │       │       └── [...]
+    │       ├── archives
+    │       │   └── archive.7z
+    │       ├── branches
+    │       ├── config
+    │       ├── description
+    │       ├── HEAD
+    │       ├── hooks
+    │       │   ├── applypatch-msg.sample
+    │       │   ├── [...]
+    │       │   └── update.sample
+    │       ├── info
+    │       │   └── exclude
+    │       ├── objects
+    │       │   ├── 05
+    │       │   │   └── 3d25959223e8173497fa7f747442b72c31671c
+    │       │   ├── 0b
+    │       │   │   └── 8d0edbf8b042998dfeb185fa2236d25dd80cf9
+    │       │   ├── [...]
+    │       │   │   └── [...]
+    │       │   ├── info
+    │       │   └── pack
+    │       ├── refs
+    │       │   ├── heads
+    │       │   │   ├── git-annex
+    │       │   │   └── master
+    │       │   └── tags
+    │       ├── ria-layout-version
+    │       └── ria-remote-ebce196a-b057-4c96-81dc-7656ea876234
+    │           └── transfer
+    ├── error_logs
+    └── ria-layout-version
+
+Beyond datasets, the RIA store only contains the directory ``error_logs``
+for error logging and the file ``ria-layout-version`` for a specification of the
```

Comment:
I have not yet talked about hashdirmixed and hashdirlower layouts in the keystore. Is there any reason for this change apart from enabling symlinked annexes? (related: Should I talk about ephemeral clones in this use case?)